internal/action/commands.go

```diff
@@ -218,8 +218,7 @@ func (s *Action) GetCommands() []*cli.Command {
 			Usage: "Search for secrets",
 			Description: "" +
 				"This command will first attempt a simple pattern match on the name of the " +
-				"secret. If that yields no results, it will trigger a fuzzy search. " +
```

Comment:
So was the fuzzy search deprecated at some point?

Reply:
We still have fuzzy search for the `gopass entry` shortcut, but it was requested that `gopass find` doesn't automatically do a fuzzy search.